spec/API_specification/examples/06_horizontal_functions.py

```diff
@@ -0,0 +1,28 @@
+"""Example of how to use a horizontal function.
+
+Horizontal functions are functions that take multiple columns as input and return a
+single column as output.
+
+Examples include:
+- `any_horizontal`
+- `all_horizontal`
+
+These can be accessed by first using ``__dataframe_namespace__`` to get the
+namespace object, and then calling the function on the namespace object and passing
+an iterable of ``Column``s as input.
+"""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from dataframe_api.typing import SupportsDataFrameAPI
+
+
+def main(df_raw: SupportsDataFrameAPI) -> SupportsDataFrameAPI:
+    df = df_raw.__dataframe_consortium_standard__(api_version="2023-11.beta")
+    ns = df.__dataframe_namespace__()
+    df = df.filter(
+        ns.any_horizontal(*[df.col(col_name) > 0 for col_name in df.column_names]),
+    )
```

Comment:
with https://github.com/data-apis/dataframe-api/pull/326, this could be simplified to

```suggestion
    df = df.filter(ns.any_horizontal(*[col > 0 for col in df.columns_iter()]))
```